### PR TITLE
Delegation info

### LINKED ISSUE #232: Add views for inspecting delegation/pool settings
**Description**

We need to support viewing information related to delegated stake and baker pool settings for accounts in the users wallet.

https://www.figma.com/file/UsjCNwB9jlgXFYPfLbZ7KZ/Desktop-Wallet?node-id=4902%3A142582

**User story**

As a user, I want to view/edit information related to my delegated stake, or my baker pool for my accounts. Furthermore, I also want to be able to view pending changes and when they take effect.

**Clarifications**
- Where are commission rates and metadata URL visible in the pool info view?

**Sub tasks**
- [x] Move actions related to baking/delegation to respective sub menu
- [x] Add view for inspecting baker pool info
- [x] Add view for inspecting delegation info
- [x] Update account balance view to also show delegated stake

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Adds views for viewing information related to baking/delegating.

Closes #232

## Changes

- Adds dedicated spaces in accounts section for baking/delegation.
- Moved baker/delegation update flows into their respective spaces
- Show delegated stake as staked on account views in addition to baker stake.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: app/features/TransactionSlice.ts [hunk 1]
```diff
@@ -717,4 +717,28 @@ export const loadingTransactionsSelector = (state: RootState) =>
 export const hasMoreTransactionsSelector = (state: RootState) =>
     state.transactions.hasMore;
 
+const isPending = (t: TransferTransaction) =>
+    t.status === TransactionStatus.Pending;
+
+export const hasPendingTransactionSelector = (
```
Comment (shjortConcordium) [inline app/features/TransactionSlice.ts:723]: I think we have a database call hasPending, for a specific account. TODO: Check this again when I'm done reviewing

Comment (shjortConcordium) [inline app/features/TransactionSlice.ts:723]: I think we have a database call hasPending, for a specific account. TODO: Check this again when I'm done reviewing

File: app/pages/Accounts/AccountDetailsPage/Baking/Baking.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+interface ActionsProps {
+    isDelegationPV: boolean;
+    disabled: boolean;
+}
+
+function Actions({ isDelegationPV, disabled }: ActionsProps) {
+    return (
+        <>
+            <ButtonNavLink
+                className="mB20:notLast flex width100"
```
Comment (shjortConcordium) [inline app/pages/Accounts/AccountDetailsPage/Baking/Baking.tsx:35]: Either move this to the css, or just remove notLast

File: app/pages/Accounts/AccountDetailsPage/MoreActions.tsx [hunk 3]
```diff
@@ -91,76 +89,47 @@ export default function MoreActions({ account, accountInfo }: Props) {
...
+            {accountHasDeployedCredentials &&
+                isDelegationPV &&
+                isDelegating &&
+                !isBaker && (
+                    <ButtonNavLink
+                        className="mB20:notLast flex width100"
+                        to={routes.ACCOUNTS_DELEGATING}
+                        disabled={!accountInfo}
+                    >
+                        Delegating
```
Comment (shjortConcordium) [inline app/pages/Accounts/AccountDetailsPage/MoreActions.tsx:130]: I'm not in love with this wording, why is it not just `Delegation`?

File: app/pages/Accounts/AccountDetailsPage/StakingDetails/StakingDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+import styles from './StakingDetails.module.scss';
+
+interface ValueProps {
+    title: string;
+    value: string;
+}
+
+function Value({ title, value }: ValueProps) {
+    return (
+        <div className="mB20:notLast">
```
Comment (shjortConcordium) [inline app/pages/Accounts/AccountDetailsPage/StakingDetails/StakingDetails.tsx:32]: love it!

File: app/pages/Accounts/AccountDetailsPage/StakingDetails/StakingDetails.tsx [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+            <Value
+                title="Target pool"
+                value={displayDelegationTarget(details.delegationTarget)}
+            />
+            <Value
+                title="Rewards wil be"
+                value={
+                    details.restakeEarnings
+                        ? 'Added to delegation amount'
+                        : 'Added to public balance'
```
Comment (shjortConcordium) [inline app/pages/Accounts/AccountDetailsPage/StakingDetails/StakingDetails.tsx:118]: lets discuss these :upside_down_face:

**Review by shjortConcordium — COMMENTED** [review]
Basically, LGTM

File: app/pages/Accounts/AccountDetailsPage/StakingDetails/StakingDetails.tsx (hunk 5 above)
Comment (shjortConcordium) [reply app/pages/Accounts/AccountDetailsPage/StakingDetails/StakingDetails.tsx:118]: Please do... change these and the baking ones to use same terminology as the transaction flows

File: app/features/TransactionSlice.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (soerenbf) [reply app/features/TransactionSlice.ts:723]: Nope you're right... filters are applied before transactions are loaded into redux.

File: app/pages/Accounts/AccountDetailsPage/StakingDetails/StakingDetails.tsx (hunk 4 above)
Comment (soerenbf) [reply app/pages/Accounts/AccountDetailsPage/StakingDetails/StakingDetails.tsx:32]: I don't know if you're sarcastic, but it makes sense here as we don't know whether it's the last in line or not 😉

File: app/features/TransactionSlice.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (soerenbf) [reply app/features/TransactionSlice.ts:723]: Ended up removing the pending stuff altogether for the staking details view.

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**